pkg/ccl/backupccl/generative_split_and_scatter_processor.go

```diff
@@ -342,8 +360,8 @@ func runGenerativeSplitAndScatter(
 
 	// TODO(pbardea): This tries to cover for a bad scatter by having 2 * the
```

Comment:
can we move this TODO with the 2* and think about whether it is still applicable? If we want to punt the thinking for later can we add your or my name in the TODO 😋

Reply:
I moved the TODO to where the factor of 2 is and put my name on it.